src/js/helpers.js

```diff
@@ -494,7 +498,7 @@ systemctl restart mender-client && \\
   "ShellCommand": "/bin/bash"
 }
 EOF
-) && systemctl restart mender-connect'
+) && systemctl restart mender-connect${setupConfirmation}'
```

Comment:
This is a bit cheating: the "Running mender setup for hosted.mender.io" message should be printed *before* calling `mender-setup`, line 491.

Also, the "hosted.mender.io" part was meant to be an example, if do onboarding with a demo om-prem installation I expect this to print something else. Maybe is better to just say "Running mender setup"

Reply:
ah right, I guess I misunderstood the reasoning behind the call... should now be more in line with the intention (I hope)